.github/actions/install-eprosima-dependencies-windows/action.yml

```diff
@@ -33,7 +33,7 @@ runs:
         cmake -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}\eprosima\install `
           -DCMAKE_PREFIX_PATH=${{ github.workspace }}\eprosima\install `
           -DTHIRDPARTY=ON -DSECURITY=ON -DCOMPILE_EXAMPLES=OFF -DEPROSIMA_BUILD_TESTS=OFF `
-          -DINTERNAL_DEBUG=ON -Ax64 -T host=x64 -B build\fastdds Fast-DDS;
+          -DINTERNAL_DEBUG=ON -DSHM_TRANSPORT_DEFAULT=OFF -Ax64 -T host=x64 -B build\fastdds Fast-DDS;
```

Comment:
:thinking: 
Wouldn't be better to deactivate it via QoS? I would also add a message (NOTE or TODO) so we remember this is done and that "we" should find a solution at some point.
Also, if not done internally to the repo, make a comment in the installation (leave a TODO if there is no documentation yet)

Reply:
Ok, I will then do it in #31